<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_iniciar">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>650</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Iniciar socket</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_apagar">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>720</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Apagar socket</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_abrir">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>680</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Abrir simulador</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>590</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Log :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_img">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>1240</width>
      <height>500</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Imagen : </string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_resultados">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>610</y>
      <width>901</width>
      <height>141</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
